compilemessages.py

```diff
@@ -0,0 +1,22 @@
+import os
```

Comment:
The language changes should be split out into a separate PR since they have no bearing on ModelService.